### PR TITLE
Added ES6 modules shortcuts

### PR DESCRIPTION
**Pull request by bouiboui**
I want to be able to do 
```js
import { requireKeys, requireParams } from '@exaprint/params-validator'
```

### CODE REVIEW

File: src/ParamsValidator.js [hunk 1]
```diff
@@ -1,37 +1,52 @@
...
-  /**
-   * @param {*} params Parameters passed
-   * @return {ParamsValidator} Fluent interface
-   */
-  static requireParams(params) {
-    for (let i = 0, keysLength = params.length; i < keysLength; ++i) {
-      if (!params[i]) {
-        throw new Error('Param #' + (i+1) + ' is missing.');
-      }
+const _extends = function (target) {
```
Comment (BasileTrujillo) [inline src/ParamsValidator.js]: Should be better in a package... or use a lodash function...

Comment (BasileTrujillo) [inline src/ParamsValidator.js]: And add comments

File: src/ParamsValidator.js [hunk 2]
```diff
@@ -1,37 +1,52 @@
...
-    return ParamsValidator;
-  }
-}
+exports.requireKeys = ParamsValidator.requireKeys;
+exports.requireParams = ParamsValidator.requireParams;
+
+exports['default'] = ParamsValidator;
 
-module.exports = ParamsValidator;
+module.exports = _extends(exports['default'], exports);
```
Comment (BasileTrujillo) [inline src/ParamsValidator.js]: Does it work if you simply export like this:
```javascript
module.exports.default = ParamsValidator;
module.exports.requireKeys = ParamsValidator.requireKeys;
module.exports.requireParams = ParamsValidator.requireParams;
```

Comment (bouiboui) [inline src/ParamsValidator.js]: Nope :( that's why I had to do this clumsy hack

**Pull request closed without merging**